src/common/xmlparser.py

```diff
@@ -1,5718 +0,0 @@
-#!/usr/bin/env python 
```

Comment:
If we remove this file, the reference to it in src/common/__init__.py + references in the documentation generation should also be removed.

Reply:
Will do. As part of building it's apparently copied to pyjmi.common.xmlparser and pymodelica.common.xmlparser, so I might need to revert this tho.